novm/manager.py

```diff
@@ -227,7 +227,7 @@ def state(output):
                 index=1+len(nics)+len(disks),
                 pci=not(nopci),
                 tag="root",
-                tempdir=self._instances.file(str(os.getpid())),
+                tempdir=self._myInstance.file("writable"),
```

Comment:
Ah I see.

I suppose a nit that I have is the manager class was intended to be able to use programatically for multiple instances.

```
   m = novm.NovmManager()
   m.create(...)
   m.run(...)
```

And I'd like it to be as stateless as possible. With the self._myInstance attribute, that possibility is gone.

Do you think we could just name the log file "%s.log" % instance_id?

That way you get the following files:

   1235.json
   1235/
   1235.log

You would just need to tweak the Nodb.file() method slightly. I think that would be preferable.

(Also, one more favor to ask -- please collapse the commits into working incremental chunks. It helps with bisecting down the road.)